dev/ProgressBar/ProgressBarAutomationPeer.cpp

```diff
@@ -18,6 +18,14 @@ ProgressBarAutomationPeer::ProgressBarAutomationPeer(winrt::ProgressBar const& o
 // IAutomationPeerOverrides
 winrt::IInspectable ProgressBarAutomationPeer::GetPatternCore(winrt::PatternInterface const& patternInterface)
 {
+    if (auto progressBar = Owner().try_as<winrt::ProgressBar>())
+    {
+        if (progressBar.IsIndeterminate())
+        {
+            return nullptr;
```

Comment:
For indeternminate progress bar, we will end up not supporting any other patterns with this return. I think what we need is to make the check inside the if block below for RangeValue pattern and not return *this if it is an indeterminate progressbar.